usage-guides/assets.md

```diff
@@ -17,6 +17,28 @@ To solve this problem we added a whitelist which you can add assets to as can be
 
 ![Whitelist asset](/images/asset_whitelist_menu.png)
 
+## Token Detection Methods
+
+There are several ways to detect tokens:
+
+**Single Account Per-Chain Detection**
+
+The best way to detect tokens for a single address is to go to blockchain & accounts balances, scroll down to the EVM accounts vuew, expand the dropdown at the account, go to the per chain tab and click on the detect tokens toggle 🔃.
+
+![Example of detecting tokens for a specific chain](/images/redetect-per-chain-view.png)
+
+**All Accounts Per-Chain Detection**
+
+Go to Account & Balances -> Blockchain Balances. Scroll down to EVM accounts and click on Re-detect tokens (🔃). Here you can select specific chains to detect tokens for all your wallets.
+
+![Example of re-detecting tokens for all EVM chains in the Blockchain Balances section.](/images/redetect-tokens-chain.png)
+
+**Dashboard Full Detection**
+
+For a complete re-detection of all tokens across all your wallets and all supported chains, use the re-detect toggle (🔃) in the main dashboard view. This will perform a full scan of your wallets on every chain.
```

Comment:
Do not conflate the world wallet with an address. Using "wallet" for an address is a mistake and misnomer used by people who don't know the difference.

```suggestion
For a complete re-detection of all tokens across all your addresses and all supported chains, use the re-detect toggle (🔃) in the main dashboard view. This will perform a full scan of your addresses on every chain but will probably take quite a long time depending on amount of tracked addresses and chains.
```